pkg/rtc/rtc.go

```diff
@@ -11,6 +11,24 @@ import (
 
 type RTC struct {
 	file *os.File
+	syscalls
+}
+
+type RTCTime struct {
+	Sec   int32
+	Min   int32
+	Hour  int32
+	Mday  int32
+	Mon   int32
+	Year  int32
+	Wday  int32
+	Yday  int32
+	Isdst int32
+}
+
+type syscalls interface {
+	ioctlGetRTCTime(int) (*RTCTime, error)
+	ioctlSetRTCTime(int, *RTCTime) error
```

Comment:
This interface should probably also be in the _linux.go file since ioctl is Linux-specific. Then the _notlinux.go would have an empty interface `type syscalls interface {}` since it it basically unused.